modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/analysis-common/70_synonyms_from_index.yml

```diff
@@ -103,7 +106,43 @@
                 query: ciao
   - match: { hits.total.value: 1 }
 
-  # Delete the synonyms set and confirm failed reload analyzers details
+---
```

Comment:
Refactored this test to separate into multiple test cases, and added an individual rule update